sdk/ml/azure-ai-ml/azure/ai/ml/constants/__init__.py

```diff
@@ -17,10 +17,13 @@
     JobType,
     NlpLearningRateScheduler,
     NlpModels,
+    TabularTrainingMode,
 )
 from ._registry import AcrAccountSku, StorageAccountType
 from ._workspace import ManagedServiceIdentityType
 
+TabularTrainingMode.__module__ = __name__
```

Comment:
to change naming form **restclient** to automl? Should this be updated for all constants?
cc: @singankit @skasturi @harneetvirk

Reply:
@novaturient95, we haven't got any response for this comment since 3 days. Should this comment be blocking for this PR or can it be a separate discussion? cc: @singankit , @skasturi , @harneetvirk